### PR TITLE
Skip saving file properties on models missing migrations required

### PR DESCRIPTION
**Pull request by adamgeorgeson**

### CODE REVIEW

**Comment by adamgeorgeson** [comment]
Need to add tests for this file. Skipped for now to fix prod issue.

File: app/uploaders/integral/image_uploader.rb [hunk 1]
```diff
@@ -97,12 +97,14 @@ def full_original_filename
 
     def store_dimensions
       return unless file && model
+      return unless model.respond_to?(:width=) && model.respond_to?(:height=)
 
       model.width, model.height = ::MiniMagick::Image.open(file.file)[:dimensions]
```
Comment (adamgeorgeson) [inline app/uploaders/integral/image_uploader.rb]: Should this be `model.image`?

Comment (patricklindsay) [reply app/uploaders/integral/image_uploader.rb]: Nah this is correct

**Comment by adamgeorgeson** [comment]
Specs have been added

**Pull request merged**